Dynatrace/dynatrace-operator
# Reworks how the CRD is generated for helm

# Description
Currently if a custom namespace is used then the conversion webhook in the CRD will have the incorrect namespace set for the `dynatrace-webhook` service. This can cause problems on the cluster.

I could not use `sed -i` because most devs in the team have MacOs which does not have GNU-sed. (failed to make `awk` handle `{{...}}`)

~**WARNING:** This is a very hacky solution, so we should think it over very carefully if we want to live with this.~

## How can this be tested?
Check that `make manifests` works and generates what we need.
Then after running`helm template dynatrace-operator ./config/helm/chart/default --namespace=testy --set installCRD=true > test.yaml`, check that in the `test.yaml` the following is present:
```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
...
  name: dynakubes.dynatrace.com
spec:
  conversion:
    strategy: Webhook
    webhook:
      clientConfig:
        service:
          name: dynatrace-webhook
          namespace: testy 
          path: /convert
      conversionReviewVersions:
      - v1beta1
  group: dynatrace.com
```


## Checklist
